boards/shields/nrf7002_ek/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* This forwards the UART1 pins to CPUNET, but as UART1 uses same pins as
+ * bucken and iovdd-ctrl, we need these pins to be controlled by the CPUAPP,
+ * so, we disable this forwarding.
+ */
+&gpio_fwd {
+	/* Disable of uart won't work, so we disable the whole node */
+	status = "disabled";
+};
```

Comment:
Please make the documentation clearer. Suggestion below

```suggestion
/* This node by default forwards the UART1 pins to CPUNET, but as UART1 uses
 * same pins as bucken and iovdd-ctrl, we need these pins to be controlled by
 * the CPUAPP. Since a child node of gpio_fwd cannot be disabled easily,
 * the entire node is disabled. If the application needs to forward other pins
 * to the CPUNET, it should create a separate instance of nrf-gpio-forwarder
 * and use it instead.
 */
&gpio_fwd {
	status = "disabled";
};
```

Reply:
Sure, go ahead and rephrase as you wish. It was only a suggestion